workflow-templates/Graphql/Rover/Graphql_Rover_K8S_Demo.yml

```diff
@@ -0,0 +1,130 @@
+name: Demo Build Workflow
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - release
+    paths-ignore:
+      - "**.md"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      AZURE_REGISTRY_NAME: crdsmlint5b5869c1
+      CRDS_ENV: demo
+      VAULT_URI: https://vault.crossroads.net:443
+      VAULT_ROLE_ID: ${{secrets.VAULT_ROLE_ID}}
+      VAULT_SECRET_ID: ${{secrets.VAULT_SECRET_ID}}
+      DOCKER_TAG: ${{github.sha}}
+      ASPNETCORE_ENVIRONMENT: demo

```

Comment:
You asked about this value. Looking at it I do not think that this value is needed for non .NET projects.